src/components/card/Card.js

```diff
@@ -1,11 +1,25 @@
 import './Card.css'
 import deceased from '../images/deceased.png'
+import arrow from '../images/arrow-u-left-top.svg'
+import { useState } from 'react'
 
 function Card(props) {
+  const [isPageTurned, setIsPageTurned] = useState(false)
+  const [showEmoji, setShowEmoji] = useState('')
```

Comment:
Maybe `[emoji, setEmoji]` would be a better state name as `showEmoji` sounds a bit like a function which is implemented.